templates/helm/Chart.yaml.tpl

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-name: {{ .ControllerName }}-chart
+name: {{ .ControllerName }}-controller
```

Comment:
i'm wondering if this will break existing controller upgrade process

Reply:
can we test it? 
it will change the name of the created chart so i believe that it will. when is the upgrade process described?